postmarketOS/pmbootstrap
# static_code_analysis.sh: Find CHANGEMEs in APKBUILDs

When the device wizard generates a `linux-*` `APKBUILD`, it puts the following line at the beginning:
> `# Kernel config based on: arch/arm/configs/(CHANGEME!)`

With this PR, Travis will check whether there are any `aports/devices/*/*` files, where the `(CHANGEME!)` has not been replaced.

### How to test
* run `test/static_code_analsysis.sh`, it should not fail (you can abort it after the APKBUILD check, when it starts checking shell scripts)
* create a new device in `pmbootstrap init`
* run the script again, it should fail and print where you need to replace something.
* clean up: delete the aports/device/*-your-new-device folders again (see `git status`)
